docs/can-guides/commitment/recipes/canvas-clock/6-min-hour-hands.js

```diff
@@ -1,8 +1,10 @@
+import { Component } from "can";
```

Comment:
I think these will need to be changed to `import { Component } from "//unpkg.com/can@5/core.mjs";`

Reply:
Done!